modules/conversion-center/assets/scss/widgets/contact-buttons-var-1.scss

```diff
@@ -9,83 +9,89 @@
 	--e-contact-buttons-message-bubble-chat-bg: #FFFFFF;
 	--e-contact-buttons-contact-text: #1C2448;
 
-	.e-contact-buttons__contact {
-		align-items: center;
-		background-color: var(--e-contact-buttons-chat-box-bg);
-		display: flex;
-		flex-direction: column;
-		gap: 20px;
-		justify-content: center;
-		padding: 0 20px 20px 20px;
-	}
-
-	.e-contact-buttons__message-bubble {
-		background-color: var(--e-contact-buttons-chat-box-bg);
-	}
-
-	.e-contact-buttons__contact-text {
-		color: var(--e-contact-buttons-contact-text);
-		font-size: 20px;
-		font-weight: 400;
-		line-height: 25px;
-		margin-bottom: 0;
-	}
-
-	.e-contact-buttons__contact-links {
-		display: flex;
-		gap: 12px;
-	}
+	.e-contact-buttons {
 
-	.e-contact-buttons__contact-icon-link {
-		align-items: center;
-		background-color: var(--e-contact-buttons-contact-button-bg);
-		border-radius: 50px;
-		display: inline-flex;
-		justify-content: center;
-		transition: $transition-hover;
-
-		svg {
-			fill: var(--e-contact-buttons-contact-button-icon);
+		&__contact {
+			align-items: center;
+			background-color: var(--e-contact-buttons-chat-box-bg);
+			display: flex;
+			flex-direction: column;
+			gap: 20px;
+			justify-content: center;
+			padding: 0 20px 20px 20px;
 		}
-
-		&:hover,
-		&:focus {
-			background-color: var(--e-contact-buttons-contact-button-bg-hover);
+	
+		&__message-bubble {
+			background-color: var(--e-contact-buttons-chat-box-bg);
+			padding: 20px;
+			padding-inline-start: 40px;
+		}
+	
+		&__contact-text {
+			color: var(--e-contact-buttons-contact-text);
+			font-size: 20px;
+			font-weight: 400;
+			line-height: 25px;
+			margin-bottom: 0;
+		}
+	
+		&__contact-links {
+			display: flex;
+			gap: 12px;
+		}
+	
+		&__contact-icon-link {
+			align-items: center;
+			background-color: var(--e-contact-buttons-contact-button-bg);
+			border-radius: 50px;
+			display: inline-flex;
+			justify-content: center;
 			transition: $transition-hover;
-
+	
 			svg {
-				fill: var(--e-contact-buttons-contact-button-icon-hover);
+				fill: var(--e-contact-buttons-contact-button-icon);
 			}
-		}
-
-		&.has-size-small {
-			height: var(--e-contact-buttons-icon-size-small);
-			width: var(--e-contact-buttons-icon-size-small);
-
-			svg {
-				height: 26px;
-				width: 26px;
+	
+			&:hover,
+			&:focus {
+				background-color: var(--e-contact-buttons-contact-button-bg-hover);
+				transition: $transition-hover;
+	
+				svg {
+					fill: var(--e-contact-buttons-contact-button-icon-hover);
+				}
 			}
-		}
-
-		&.has-size-medium {
-			height: var(--e-contact-buttons-icon-size-medium);
-			width: var(--e-contact-buttons-icon-size-medium);
-
-			svg {
-				height: 28px;
-				width: 28px;
+	
+			&.has-size-small {
+				height: var(--e-contact-buttons-icon-size-small);
+				width: var(--e-contact-buttons-icon-size-small);
+	
+				svg {
```

Comment:
@mserino just a thought here as I'm not overly familiar with this widget, but can the SVG height & width use the same custom properties e.g. `--e-contact-buttons-icon-size-small` or perhaps be set once as height: 100%, width: 100%; so match the container rather than resetting with each size variation?

Reply:
Yeah I'm hardcoding the values since they will always be depending on the size of the button, and they're decided by design. It didn't make much sense to create variables as they'll always have those size and they're fixed.